src/lua/loaders.lua

```diff
@@ -250,6 +250,47 @@ table.insert(package.loaders, 5, gen_loader_func(search_rocks_lib, load_lib))
 -- package.cpath  7
 -- croot          8
 
+-- Search for modules next to the main script.
+local script = minifio.script()
+if script ~= nil and script ~= '-' then
+    -- It is important to obtain the directory at initialization,
+    -- before any cwd change may occur. The script path may be
+    -- passed as relative to the current directory.
+    local script_dir = minifio.dirname(minifio.abspath(script))
+    local function script_dir_fn()
+        return script_dir
+    end
+
+    -- Search non-recursively, only next to the script.
+    local search_app_lua = gen_search_func(script_dir_fn, LUA_TEMPLATES)
+    local search_app_lib = gen_search_func(script_dir_fn, LIB_TEMPLATES)
+    local search_app_rocks_lua = gen_search_func(script_dir_fn,
+        ROCKS_LUA_TEMPLATES)
+    local search_app_rocks_lib = gen_search_func(script_dir_fn,
+        ROCKS_LIB_TEMPLATES)
+
+    -- Mix the script directory loaders into corresponding
+    -- searchroot based loaders. It allows to don't change
```

Comment:
Minor: I suggest rephrasing it like the following:
`It allows to avoid changing ordinals of the loaders ...`
The original comment is hard for me to read:(.

Reply:
No problems, changed.